implementations/c/ockam/channel/tests/channel_test.c

```diff
@@ -71,7 +71,7 @@ ockam_error_t parse_opts(int argc, char* argv[])
 
 int main(int argc, char* argv[])
 {
-  ockam_error_t                    error             = OCKAM_ERROR_NONE;
+  int                              status            = 0;
```

Comment:
Should status have its own `_t` struct?

Reply:
The problem is that now errors are defined by error code and error domain (string that indicates which module returned the error). I think creating separate error domains and codes for tests is overkill (as well as adding separate error codes for tests to our non-test code, which was the case somewhere before this PR). We need to improve tests in many ways, particularly add logs, for errors I decided to just stick with simplest way to make it work. Further work IMHO goes out of scope of this PR.

Reply:
Okay, should we add it as furture work?